wasp/widgets.py

```diff
@@ -246,6 +259,10 @@ def draw(self):
         color = self._color
         light = self._lowlight
 
+        if not self._read_theme:
+            self.__init__(self._steps, self._x, self._y, wasp.system.theme('slider-default'))
+            self._read_theme = True
+
```

Comment:
self._lowlight also needs to be set None in `__init__()` since we can't calculate the lowlight colour until we've decided what the highlight colour is going to be.